poetry.toml

```diff
@@ -0,0 +1,3 @@
+[virtualenvs]
+in-project = false
+create = false
```

Comment:
good shit. didn't know about this.

Reply:
Yes! I don't like poetry managing python environments, and this works for me. Another way is to set it globally, or with environment variables.